doc/whats-new.rst

```diff
@@ -75,6 +75,7 @@ Bug fixes
   ``DataArray`` objects, previously only the global attributes were retained (:issue:`4497`, :pull:`4510`).
   By `Mathias Hauser <https://github.com/mathause>`_.
 - Improve performance where reading small slices from huge dimensions was slower than necessary (:pull:`4560`). By `Dion Häfner <https://github.com/dionhaefner>`_.
+- Fix bug where ``dask_gufunc_kwargs`` where silently changed in :py:func:`apply_ufunc` (:pull:`4576`). By `Kai Mühlbauer <https://github.com/kmuehlbauer>`_.
```

Comment:
:grimacing: Now I have a typo here, second `where` should be `were` or better `was`. @max-sixty Is it appropriate to add [ci-skip] for such small change to not run CI again?

Reply:
OK, so it won't work. Maybe because this is triggered via a PR?

Reply:
I think you need to use `[skip ci]` or `[ci skip]` (` ` instead of `-`) ([docs](https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml#ci-triggers))

Reply:
@keewis Yes, that's probably the case. Thanks for letting me know!